extra/1-currency-conversion.js

```diff
@@ -15,7 +17,10 @@ function convertToUSD() {}
   They have also decided that they should add a 1% fee to all foreign transactions, which means you only convert 99% of the £ to BRL.
 */
 
-function convertToBRL() {}
+function convertToBRL(price) {
+  return Math.round((price * 0.99) * 5.7 * 100) / 100;
```

Comment:
Hi Alena, I like how clean your code looks, I find it easy to read and understand!
I'm curious, could you help me understand why this is multiplied by 100 then divided by 100?